sympy/core/sympify.py

```diff
@@ -51,6 +51,45 @@ class CantSympify(object):
     pass
 
 
+def convert_numpy_types(a):
+    """
+    Converts a numpy datatype input to an appropriate sympy type.
+
+    Examples
+    ========
+
+    >>> import sympy as sp
+    >>> import numpy as np
+
+    >>> sp.Float(np.int64(5))
+    5.00000000000000
+
+    >>> sp.Float(np.float128(5))
+    5.00000000000000000
+
+    >>> sp.Float(np.float32(5))
+    5.00000
+
+    >>> x = sp.symbols('x')
+    >>> print(x * np.int64(1))
+    x
+    """
```

Comment:
I don't think this last docstring adds anything useful.

Reply:
Removed.